### PR TITLE
include classes which not used in game

### PR DESCRIPTION
**Pull request by TheLeerName**
hscript: 📈

for example u cant `addHaxeLibrary('FlxGlitchEffect', 'flixel.addons.effects.chainable')` if source code not used this class

previous pr was for experimental branch: #12733

### CODE REVIEW

**Review by moxie-coder — APPROVED** [review]

File: source/macros/AdditionalClasses.hx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+			"Lambda",
+			"StringBuf",
+		];
+
+		var excludePackages:Array<String> = [
+			"flixel.addons.editors.spine",
+			"flixel.addons.nape",
+			"flixel.system.macros",
+
+			"haxe.macro",
```
Comment (ActualMandM) [inline source/macros/AdditionalClasses.hx:38]:
```suggestion
			"haxe.macro",
			#if (js || hxcpp) "haxe.atomic.AtomicObject", #end
```

**Review by RagdollClash — CHANGES_REQUESTED** [review]
This currently runs into this error on compilation: `D:\Programs\HaxeToolkit\haxe\std/haxe/atomic/AtomicObject.hx:8: characters 8-58 : JavaScript and Hxcpp do not support AtomicObject`

**Review by RagdollClash — APPROVED** [review]

**Comment by TheLeerName** [comment]
omg sscript broken

**Comment by TheLeerName** [comment]
#12953

**Pull request closed without merging**